docs/language/README.md

```diff
@@ -2,6 +2,11 @@
 
 The language documents:
 * provide an [overview](overview.md) of the Zed language,
-* layout some [conventions](conventions.md) for the documenation, and
+* describe Zed's [dataflow model](dataflow-model.md),
+* show the syntax of [statements](statements.md) that define constants, functions, and named types,
+* explain how Zed [data types](data-types.md) are used,
+* describe [expression](expressions.md) syntax, including for [search](search-expressions.md),
+* explain how [lateral queries](lateral-subqueries.md) are used,
+* describe the the concepts of [shaping and type fusion](shaping.md), and
```

Comment:
It felt awkward to write such a long extension of this narrative reveal of hyperlinks, so if folks wanna go another direction here, I'm open to that.